index.js

```diff
@@ -4361,7 +4361,7 @@
   //. inputs! [MDN][date parsing] warns against using the `Date` constructor
   //. to parse date strings:
   //.
-  //. > __Note:__ parsing of date strings with the `Date` constructor \[…] is
+  //. > __Note:__ parsing of date strings with the `Date` constructor […] is
```

Comment:
Considerable time and energy was expended to enable this change. :rofl:

Thanks for helping to make it happen, @wooorm.